unlock-app/app/frames/checkout/route.tsx

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable react/jsx-key */
+import React from 'react'
+import { Button } from 'frames.js/next'
+import { frames } from './frames'
+import { getDefaultFrame } from './components/defaultFrame'
+import { TransactionSuccess } from './components/TransactionSuccess'
+import { getLockDataFromCheckout } from './components/utils'
+
+const getInitialFrame = frames(async (ctx) => {
+  const id = ctx.searchParams.id
+  const state = ctx.state
+
+  const lock = await getLockDataFromCheckout(id)
+  state.lock = lock
+
+  return getDefaultFrame(state)
+})
+
+const getOtherFrames = frames(async (ctx) => {
+  const state = ctx.state
+  const lock = state.lock!
+  const success = ctx.searchParams.success
+
+  if (success === 'true') {
+    return {
+      image: <TransactionSuccess lock={lock} />,
+      buttons: [
+        <Button action="link" target="https://app.unlock-protocol.com/keychain">
+          View on keychain
+        </Button>,
+        <Button action="tx" target="/txdata" post_url="?success=true">
+          Buy again
+        </Button>,
```

Comment:
Let's remove this since it is usually not possible to buy again!

Reply:
If the checkotu as a redirect value, let's just add a button to go there (the checkout config may also have a CTA)